src/screens/poa-form/index.js

```diff
@@ -251,9 +282,12 @@ class PoAForm extends React.Component {
           data-address-type={'street_address'}
           placeHolder="Street Address"
         />
-        {errors && this.state.errors[`${name}_locality`] ? (
+        {errors && this.state.errors[name][`${name}_locality`] ? (
```

Comment:
Can you make an error component that accepts some configuration and renders the message if the error is present?

Reply:
👍